model/atmosphere/diffusion/tests/diffusion_stencil_tests/test_calculate_nabla2_of_theta.py

```diff
@@ -44,15 +43,14 @@ def reference(
 
     @pytest.fixture
     def input_data(self, grid):
-        z_nabla2_e = data_alloc.random_field(grid, dims.EdgeDim, dims.KDim, dtype=wpfloat)
-        geofac_div = data_alloc.random_field(grid, dims.CellDim, dims.C2EDim, dtype=wpfloat)
-        geofac_div_new = data_alloc.as_1D_sparse_field(geofac_div, dims.CEDim)
+        z_nabla2_e = data_alloc.random_field(grid, dims.EdgeDim, dims.KDim, dtype=ta.wpfloat)
+        geofac_div = data_alloc.random_field(grid, dims.CEDim, dtype=ta.wpfloat)
```

Comment:
as far as I am concerned, this is fine since it's a stencil test. However, we could still use `flatten_first_two_dims` so that it matches the other places where we process `geofac_div`

Reply:
For the stencil tests, I switched to directly allocate the `CEDim` everywhere, because the data is random, so we have no pre-existing data that needs to be reshaped, and it is very probably that the `flatten_first_two_dims` does more than a simple reshape on metadata but moves around some data, so it does only consume time for the tests.